core/shared/platform/common/posix/posix_time.c

```diff
@@ -4,6 +4,43 @@
  */
 
 #include "platform_api_vmcore.h"
+#include <time.h>
+
+uint64
+convert_timespec(const struct timespec *ts)
```

Comment:
to avoid duplicating the implementation from the posix.c file, maybe we can create a utility file in the `core/shared/utils` (let's call it bh_time.c / bh_time.h) so it's available both in posix_time.c and posix.c file.